backend/copr_backend/storage.py

```diff
@@ -171,6 +227,11 @@ def upload_build_results(self, chroot, results_dir, target_dir_name):
 
                 self.log.info("Uploaded to Pulp: %s", path)
 
+        data = {"artifacts": artifacts}
+        path = os.path.join(results_dir, "pulp.json")
+        with open(path, "w+", encoding="utf8") as fp:
```

Comment:
unfortunately I've got empty artifacts every time in the uploaded build results :/
e.g. https://download.copr-dev.fedorainfracloud.org/results/nikromen/test-pulp-2/fedora-39-x86_64/02918749-python-dside/pulp.json
https://download.copr-dev.fedorainfracloud.org/results/nikromen/test-pulp-2/fedora-40-x86_64/02918747-argparse-manpage/pulp.json

Reply:
Good catch, I probably incorrectly rebased on top of PR #3391
Fixed